connector_infor_account_move/models/account_invoice.py

```diff
@@ -0,0 +1,117 @@
+      </ActionCriteria>
+    </Process>
+    <SourceSystemJournalEntry>
+      <JournalEntryHeader>
+        <JournalID accountingEntity="${BUSINESS_UNIT}" variationID="">{{INVOICE_ID}}</JournalID>
+        <DisplayID>{{INVOICE_ID}}</DisplayID>
+        <BaseJournalEntryHeader>
+          <LedgerID>A</LedgerID>
+          <LedgerType>Actual</LedgerType>
+          <AccountingJournalReference>
+            <ID>{{JOURNAL_CODE}}</ID>
+          </AccountingJournalReference>
+          <LedgerSetupReference>
+            <DocumentID>
+              <ID>A</ID>
+            </DocumentID>
+            <BookInstanceID>A</BookInstanceID>
+          </LedgerSetupReference>
+          <JournalStatus>
+            <Code>Final</Code>
+          </JournalStatus>
+        </BaseJournalEntryHeader>
+      </JournalEntryHeader>
+      <JournalEntryLine sequence="${JOURNAL_ENTRY_SEQUENCE}">
+      {% for line in JOURNAL_LINES %}
+        <Amount currencyID="{{ line.company_id.currency_id.name }}">{{SEC_AMOUNT}}</Amount>
+        <FunctionalAmount currencyID="{{COMPANY_CURRENCY}}">{{COMPANY_AMOUNT}}</FunctionalAmount>
+        <ReportingCurrencyAmount currencyID="{{CURRENCY}}">0</ReportingCurrencyAmount>
+        <GLAccount>
+          <GLNominalAccount>{{ line.account_id.code }}</GLNominalAccount>
+          <AccountingChartReference>
+            <ID>${BUSINESS_UNIT}</ID>
+          </AccountingChartReference>
+        </GLAccount>
+        <DimensionCodes>
+          <DimensionCode sequence="1" listID="${CUSTOM_ATTRIBUTE}">${CUSTOM_VALUE}</DimensionCode>
+        </DimensionCodes>
+        {%- if line.credit > 0 -%}
+            <DebitCreditFlag>C</DebitCreditFlag>
+        {% else %}
+            <DebitCreditFlag>D</DebitCreditFlag>
+        {%- endif %}
+        <SourceAccountingDate>
+          <PeriodID>${FISCAL_PERIOD}</PeriodID>
+          <Year>${FISCAL_YEAR}</Year>
+          <PostDateTime>{{ line.create_date }}</PostDateTime>
+        </SourceAccountingDate>
+        <TransactionReferenceText>{{INVOICE_NUMBER}}</TransactionReferenceText>
+        <Description languageID="${LANGUAGE}">${DESCRIPTION}</Description>
+        <MemoAmount>
+          <AmountNumeric />
+        </MemoAmount>
+        <ExchangeRateDateTime>${TRANSACTION_DATE}</ExchangeRateDateTime>
+        <UserArea>
+          <Property sequence="1">
+            <NameValue name="${CUSTOM_ATTRIBUTE}" type="String" unitCode="EA">${CUSTOM_VALUE}</NameValue>
+          </Property>
+        </UserArea>
+      {% endfor %}
+      </JournalEntryLine>
+    </SourceSystemJournalEntry>
+  </DataArea>
+</ProcessSourceSystemJournalEntry>
+"""
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+
+    @api.multi
+    def invoice_validate(self):
+        template = Template(body_template)
+        today = fields.Datetime.now(pytz.timezone('GMT'))
+        for invoice in self:
+            # Only applicable for Vendor bills
+            if invoice.type == 'in_invoice':
```

Comment:
@bodedra @guewen it's applicable for any posted journal entry in the journal configured on the backend.